.vscode/settings.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
added the same settings config to this repo so that it matches what's going on in the monorepo and we don't get linter whiplash if working directly in submodule?